DataFormats/HcalDigi/src/classes_def.xml

```diff
@@ -42,11 +42,6 @@
    <class name="HcalTTPDigi" ClassVersion="10">
     <version ClassVersion="10" checksum="3127055275"/>
    </class>
-   <class name="HcalUpgradeDataFrame" ClassVersion="12">
-    <version ClassVersion="10" checksum="2143388425"/>
```

Comment:
Do we have any samples with HcalUpgradeDataFrame produced, which may be needed for future (re)processing or analysis? (the question is to any sample that possibly had HcalUpgradeDataFrame produced and stored empty)
Complete removal of dictionaries and the type will lead to crashes and requirement to add drop statements.